packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/proxies/documentservice/DocumentProxy.java

```diff
@@ -9,7 +9,7 @@
 import javax.ws.rs.*;
 import javax.ws.rs.core.MediaType;
 
-@RegisterRestClient(configKey = "xdd-document-service")
+@RegisterRestClient(configKey = "xdd-dev-service")
```

Comment:
Once this PR is merged you should post in the askem-hmi channel advising people that we're temporarily moving back to the dev instance of xdd and that search results may change as a result. You can also let people know that on their dev instance they've made performance improvements to the unified endpoint so they'll see that too

Reply:
Will do :)